Signal/src/ViewControllers/MediaPageViewController.swift

```diff
@@ -108,6 +110,26 @@ class MediaPageViewController: UIPageViewController, UIPageViewControllerDataSou
     var currentCaptionView: CaptionView!
     var pendingCaptionView: CaptionView!
 
+    // MARK: ImageRail
+
+    var galleryRailView: GalleryRailView!
```

Comment:
Is the intention to re-use this rail class for the attachment approval changes?

Reply:
It's not clear to me yet how much will be reusable. Originally the designs were quite different so most of the complicated code wasn't relevant. I'll try to re-use where it makes sense.